CHANGELOG.md

```diff
@@ -289,7 +289,7 @@
 
 ### Fixed
 
--   Fix build problems on windows [Level#247](https://github.com/Level/leveldown/issues/247) ([**@obastemur**](https://github.com/obastemur))
```

Comment:
The reason I missed these is because they were plain urls and not in the `#NR` format.